docker-compose.yml

```diff
@@ -18,6 +18,12 @@ services:
       - db
     links:
       - db:db
+    healthcheck:
+        test: ["CMD", "curl", "-f", "http://localhost:3000"]
+        interval: 1m30s
+        timeout: 10s
+        retries: 3
+        start_period: 40s
```

Comment:
I feel these values are a bit high 🤔 does it really take 40s for the server to start?

Reply:
i use default values, not sure which are the average, maybe 20 seconds? @JulianPasquale

Reply:
where did you take the defaults from?

Reply:
https://docs.docker.com/reference/compose-file/services/#healthcheck